Sources/StreamChat/Models/MessageReaction.swift

```diff
@@ -19,9 +19,25 @@ public struct ChatMessageReaction: Hashable {
     /// The date the reaction was last updated.
     public let updatedAt: Date
     
-    /// Custom data
-    public let extraData: [String: RawJSON]
-    
     /// The author.
     public let author: ChatUser
+
+    /// Custom data
+    public let extraData: [String: RawJSON]
+
+    public init(
+        type: MessageReactionType,
+        score: Int,
+        createdAt: Date,
+        updatedAt: Date,
+        author: ChatUser,
+        extraData: [String: RawJSON]
```

Comment:
Hm, why is it `public`? 🤔

Reply:
We need this for testing. Also, the customer might need it as well. I didn't saw much value on creating an internal init just for testing